README.md

```diff
@@ -15,7 +15,7 @@ Part 3: [Final touches](https://github.com/microverseinc/curriculum-javascript/b
 
 
 # screenshot 
-will be added
+![screenshot](/src/images/Demo.jpg)
 
 ## Built using
 
```

Comment:
[OPTIONAL] The MIT link is broken, I would recommend fixing that so your project is documented professionally.